app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/OnboardRouterActivityKt.kt

```diff
@@ -0,0 +1,212 @@
+package com.mapbox.services.android.navigation.testapp.activity
+
+import android.os.Bundle
+import android.os.Environment
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.geojson.Point
+import com.mapbox.mapboxsdk.annotations.MarkerOptions
+import com.mapbox.mapboxsdk.camera.CameraUpdateFactory
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.maps.MapboxMap
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.navigation.base.route.DirectionsSession
+import com.mapbox.navigation.base.route.Router
+import com.mapbox.navigation.base.route.model.Route
+import com.mapbox.navigation.base.route.model.RouteOptionsNavigation
+import com.mapbox.navigation.directions.session.MapboxDirectionsSession
+import com.mapbox.navigation.route.onboard.MapboxOnboardRouter
+import com.mapbox.navigation.route.onboard.model.Config
+import com.mapbox.navigation.utils.extensions.ifNonNull
+import com.mapbox.services.android.navigation.testapp.R
+import com.mapbox.services.android.navigation.testapp.utils.Utils
+import com.mapbox.services.android.navigation.ui.v5.route.NavigationMapRoute
+import com.mapbox.services.android.navigation.v5.utils.extensions.mapToDirectionsRoute
+import com.mapbox.turf.TurfConstants
+import com.mapbox.turf.TurfMeasurement
+import java.io.File
+import java.util.ArrayList
+import kotlinx.android.synthetic.main.activity_mock_navigation.*
+import timber.log.Timber
+
+class OnboardRouterActivityKt : AppCompatActivity(),
+    OnMapReadyCallback,
+    MapboxMap.OnMapClickListener,
+    DirectionsSession.RouteObserver {
+
+    private lateinit var offboardRouter: Router
```

Comment:
I believe it should be named as `onboardRouter` or just `router`

Reply:
yep, you're right